src/variables/components/CreateVariableOverlay.tsx

```diff
@@ -9,23 +9,28 @@ import GetResources from 'src/resources/components/GetResources'
 
 // Types
 import {ResourceType} from 'src/types'
+import ErrorBoundary from '../../shared/components/ErrorBoundary'
+import {ErrorHandling} from '../../shared/decorators/errors'
 
 type Props = RouteComponentProps<{orgID: string}>
 
+@ErrorHandling
 class CreateVariableOverlay extends PureComponent<Props> {
   public render() {
     return (
       <Overlay visible={true}>
         <Overlay.Container maxWidth={1000}>
-          <Overlay.Header
-            title="Create Variable"
-            onDismiss={this.handleHideOverlay}
-          />
-          <Overlay.Body>
-            <GetResources resources={[ResourceType.Variables]}>
-              <VariableFormContext onHideOverlay={this.handleHideOverlay} />
-            </GetResources>
-          </Overlay.Body>
+          <ErrorBoundary>
```

Comment:
The error decorator and error boundary seem redundant

Reply:
Tested to make sure that just the error boundary was sufficient in this case and it turned out to be.  Since the decorator only works on class components and the error boundary works on both I kept the error boundary and removed the decorator.  In testing I realized the user will get a better experience with a boundary a bit further in so I made that change at the same time.